extensions/amp-story/1.0/amp-story-quiz.js

```diff
@@ -368,6 +374,135 @@ export class AmpStoryQuiz extends AMP.BaseElement {
   updateQuizToPostSelectionState_(selectedOption) {
     this.quizEl_.classList.add('i-amphtml-story-quiz-post-selection');
     selectedOption.classList.add('i-amphtml-story-quiz-option-selected');
+
+    if (this.responseData_) {
+      this.quizEl_.classList.add('i-amphtml-story-quiz-has-data');
+    }
+  }
+
+  /**
+   * @private
+   */
+  updateOptionPercentages_() {
+    if (!this.responseData_) {
+      return;
+    }
+
+    const options = toArray(
+      this.quizEl_.querySelectorAll('.i-amphtml-story-quiz-option')
+    );
+
+    const percentages = this.preprocessPercentages_(this.responseData_);
+
+    this.responseData_['responses'].forEach(response => {
+      options[response['reactionValue']].querySelector(
+        '.i-amphtml-story-quiz-percentage-text'
+      ).textContent = `${percentages[response['reactionValue']]}%`;
```

Comment:
Re: a `TODO` for i18n, the `%` sign can be preceded by any kind of space, or even appear before the number.

https://en.wikipedia.org/wiki/Percent_sign#Form_and_spacing